pyproject.toml

```diff
@@ -77,7 +77,8 @@ env = [
   "FS_UPLOAD_CONFIG__ROOT=../upload",
   "FS_UPLOAD_CONFIG__MKDIR=true",
   "FS_DOWNLOAD_CONFIG__PROTOCOL=file",
-  "ENV=TEST"
+  "ENV=TEST",
+  "MAIL_API_URL=http://mail-api:8765/internal/confirmation/send"
```

Comment:
This conflicts with MAIL_API added to sbl-project.  Will need to either update that one to MAIL_API_URL, or change these to MAIL_API.

Reply:
Created [sbl-project #304](https://github.com/cfpb/sbl-project/issues/304) for this.